CRM/Case/Form/Task.php

```diff
@@ -40,6 +40,13 @@ class CRM_Case_Form_Task extends CRM_Core_Form_Task {
   // Must be set to entity shortname (eg. event)
   static $entityShortname = 'case';
 
+  /**
+   * Must be set to queryMode
+   *
+   * @var int
+   */
+  static $queryMode = CRM_Contact_BAO_Query::MODE_CASE;
```

Comment:
Where is this used?

Reply:
I couldn't find any instances?

Reply:
https://github.com/civicrm/civicrm-core/pull/12316/files#diff-8e58e0ef24ffee40975a92a797f7acecR135

It's a baby step towards making that preProcessCommon function generic enough to handle more than just the case task.